releasenotes/notes/cvar-expectation-bbc2ccff95132b35.yaml

```diff
@@ -0,0 +1,52 @@
+---
+features:
+  - |
+    CVaR Expectation Values
+
+    Given a diagonal observable H, often corresponding to the objective function of an optimization problem, we are
```

Comment:
Is any of this text useful explanation/background for in the class itself too? I am not suggesting removing it from here just that perhaps having it become part of the main documentation too, which it would were it in the class docstring.

Reply:
Yes, that's a good idea. Added in 3d47faf